hushline/__init__.py

```diff
@@ -54,6 +56,19 @@ def handle_rate_limit_exceeded(e):
     for module in [admin, settings]:
         app.register_blueprint(module.create_blueprint())
 
+    # Correctly initialize the debug_cli AppGroup before defining commands
+    debug_cli = AppGroup("debug", help="Debugging related commands")
```

Comment:
This line plus the `app.cli.add_command(debug_cli)` can just be done like this:

```python
@app.cli.group(help="Debugging related commands")
def debug() -> None:
    pass
```

This is how the docs recommend it, and it seems to be preferrable, but this is just a nit.

Reply:
New update pushed